tests/test_evaluation/test_split_evaluation.py

```diff
@@ -305,3 +295,70 @@ def test_multilingual_one_missing_lang_in_one_split(
     # output merged result with previous results
     assert results[0].scores.keys() == {"test", "val"}
     assert len(results[0].scores["test"]) == 2
+
+
+def test_all_splits_evaluated_with_overwrite(model, tasks, tmp_path):
```

Comment:
What does "all splits" mean here? There are test and val but only val is run here. 
>because all splits should be run with overwrite

Based on this comment, is it correct that "all" means all specified splits in the run method, and not all splits available in the task metadata's eval_splits?

Perhaps this can be spelled out, or preferably shown in additional test cases, where the 1st and 2nd runs have different `eval_splits`

Reply:
> means all specified splits in the run method,

Yes, all splits specified in `evaluation.run`